doc/guides/build/kconfig/setting.rst

```diff
@@ -167,6 +167,22 @@ The application configuration can come from the sources below. By default,
 6. Otherwise, :file:`prj.conf` is used if it exists in the application
    directory
 
+Per default all configuration files will be taken from the application's
+configuration directory except for files with an absolute path that are
+given with the ``CONF_FILE`` argument.
+
+The application's configuration directory will be defined as:
+
+1. If ``APPLICATION_CONFIG_DIR`` is specified by the user with
+   ``-DAPPLICATION_CONFIG_DIR=<path>`` or in a CMake file before
+   ``find_package(Zephyr)`` then this folder is used a the application's
+   configuration directory.
+
+2. If the folder :file:`boards/<BOARD>` exists, then this folder will be
+   selected as the application's configuration directory.
+
+3. The application's source directory.
```

Comment:
Each of the three options for how an application configuration directory can be set should not mention that they are setting the application configuration directory.

At least, either all of them, or none of them should mention it. Now it is two of three.